extensions/wikia/Wall/WallHistoryController.class.php

```diff
@@ -282,6 +282,8 @@ private function getFormatedHistoryData( $history, $threadId = 0 ) {
 			$userid = $user->getId();
 			if ( $user->isAnon() ) WallRailController::addAnon( $userid, $user );
 			else WallRailController::addUser( $userid, $user );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add curly braces around the nested statement(s). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)